parsl/dataflow/dflow.py

```diff
@@ -280,7 +280,7 @@ def handle_update(self, task_id, future, memo_cbk=False):
                 final_state_flag = True
                 self.tasks_failed_count += 1
 
-                self.tasks[task_id]['time_completed'] = str(datetime.now().strftime('%Y-%m-%d %H:%M:%S'))
+                self.tasks[task_id]['time_completed'] = time.time()
```

Comment:
I vote to change `time_completed` to `time_returned`, to distinguish when the task finished running on the worker from when the task was returned to the DFK. Soon we should add a `time_completed` which will refer to when the task finished running on the worker.

Reply:
That makes sense and I will change this in the code.